### ISSUE TITLE
Media relations - networking (NB: connected to #14)

### ISSUE THREAD

**Issue by prestonaustin** [therabble/sustain-care.org#21]
`
    Issue ID:    021
    Issue Name:    Media relations - networking (NB: connected to Issue 014 #14)
    Description:    Leveraging existing/building new connections with local media to promote initiative. Begin in earnest once we have a website to point people/pilot program in place.
    Tags:    promotion

`

@ramlyed I think we want to avoid issues that are _ongoing work_ and turn them into issues that are _one result of that work_ so this one maybe is entirely redundant with #14 but I'm leaving it here in case the doc over in google has useful stuff in the comments, and for completeness.

https://docs.google.com/document/d/1h9k5oVXKIwElhNLgJhI4uWFOQJjJraaZnlINaiz-n4A/edit#heading=h.82wmb7jlu9rf

**Comment by ramlyed** [comment]
Yes agreed. I'll think of what instructions are needed in Documentation to prevent such issues of being created
